common/Reducers.jsm

```diff
@@ -466,7 +466,13 @@ function DiscoveryStream(prevState = INITIAL_STATE.DiscoveryStream, action) {
     case at.DISCOVERY_STREAM_LAYOUT_UPDATE:
       return {...prevState, lastUpdated: action.data.lastUpdated || null, layout: action.data.layout || []};
     case at.DISCOVERY_STREAM_LAYOUT_RESET:
-      return {...prevState, lastUpdated: INITIAL_STATE.DiscoveryStream.lastUpdated, layout: INITIAL_STATE.DiscoveryStream.layout};
+      return {
+        ...prevState,
```

Comment:
I wonder if it actually might be better to do:
```js
return {...INITIAL_STATE, config: prevState.config}
``
Since the intention is for everything to be reset except the config. What do you think?

Reply:
Works for me. Should do exactly the same thing.